tests/helpers_for_tests.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2022, Optimizely
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http:#www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from requests import Response
+from typing import Optional
+
+
+def fake_server_response(status_code: int = None, content: bytes = None, url: str = None) -> Optional[Response]:
```

Comment:
@andrewleap-optimizely What do you think here, I got content as type bytes, correct?

Reply:
Also @andrewleap-optimizely I'm not happy with the file name "helpers_for_tests". There is helpers in optimizely folder, so wanted to be consistent, but can't start the file with prefix test_ (as in test_helpers). Could use testutils.py, utils.py... suggestions?

Reply:
> @andrewleap-optimizely What do you think here, I got content as type bytes, correct?

Changed bytes to str

Reply:
What about adding it as a method to `BaseTest` in base.py? Then it can be called easily with `self.fake_server_response` in any test